src/init/command.ts

```
@@ -1704,6 +1724,109 @@ function getUpdaterInstallBlocker(dependencies: Map<string, string>, packageMana
+  if (logSelection)
+    pLog.info(`@capacitor/core version is ${coreVersion}, installing latest capacitor-updater`)
+  return { versionToInstall: 'latest', shouldOfferDirectInstall: true }
+}
+
+function getUpdaterInstallCommand(pm: PackageManagerInfo, versionToInstall: string, force = false): string {
+  const forceFlag = force ? ' --force' : ''
+  return `${pm.installCommand}${forceFlag} ${CAPGO_UPDATER_PACKAGE}@${versionToInstall}`
+}
+
+function runUpdaterInstallCommand(pm: PackageManagerInfo, packageJsonPath: string, versionToInstall: string): void {
+  const [command, ...args] = pm.installCommand.split(/\s+/).filter(Boolean)
```

ESLint

[error] Move this regular expression to module scope to avoid re-compilation on every call.

(e18e/prefer-static-regex)